packages/common/src/hooks/useDrawer/useDrawer.ts

```diff
@@ -26,6 +27,13 @@ export const useDrawer = create<DrawerController>(set => {
     close: () => set(state => ({ ...state, isOpen: false, hoverOpen: false })),
     toggle: () =>
       set(state => ({ ...state, isOpen: !state.isOpen, hasUserSet: true })),
+    onClick: () =>
```

Comment:
this was because the AppNavLink only knows the current state if it re-renders, and I didn't see the need to increase the number of renders just to get the state.